src/pages/ActiveTasks/ActiveTasks.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ActiveTasks from "./ActiveTasks";
+import tasks from "../../MockData/tasks";
+import { MemoryRouter } from "react-router-dom";
+
+describe("ActiveTask page component", () => {
+  it("should render the ActiveTask page", () => {
+    render(
+      <MemoryRouter>
+        <ActiveTasks />
+      </MemoryRouter>
+    );
+
+    const activeTask = screen.getByTestId("task-page");
+    expect(activeTask).toBeInTheDocument();
+  });
+
+  it("should render the header component", () => {
+    render(
+      <MemoryRouter>
+        <ActiveTasks />
+      </MemoryRouter>
+    );
+
+    const header = screen.getByText("WAY OF THE VIKING");
+    expect(header).toBeInTheDocument();
+    console.log(header);
+  });
+
+  it("should render the navigation component", () => {
+    render(
+      <MemoryRouter>
+        <ActiveTasks />
+      </MemoryRouter>
+    );
+
+    const home = screen.getByText("Home");
+    const tasks = screen.getByText("Tasks");
+    const calendar = screen.getByText("Calendar");
+    const leaderboard = screen.getByText("Leaderboard");
+    expect(home).toBeInTheDocument();
+    expect(tasks).toBeInTheDocument();
+    expect(calendar).toBeInTheDocument();
+    expect(leaderboard).toBeInTheDocument();
+  });
+
+  it("should check the initial state of the button", () => {
+    render(
+      <MemoryRouter>
```

Comment:
Instead of using `MemoryRouter` import the custom render function and use that, please.